docs/overview/README.md

```diff
@@ -0,0 +1,58 @@
+
+One App helps solve this problem by providing a framework to build quality, modularized web applications. It helps you break the "Frontend Monolith" and split your application via independently developed, tested, and deployed **Micro Frontends** in the form of **Holocron Modules**.
+
+# Benefits
+
+The benefits to using One App include:
+
+- **Micro Frontends:** Powered by Holocron, One App allows for code splitting via independently developed, tested, and deployed Modules.
+- **Decoupled Codebases:** Each Holocron module has its own repository, reducing large and overwhelming codebases and helping developers to understand the code better. This also avoids accidental coupling by setting specific boundaries.
+- **Security:** Configurable security settings, all pages are protected by a content security policy and all modules are injected with sub-resource integrity.
+- **Flexibility:** An omnichannel solution to UI. Whether you're performing serverside, clientside, delivering modules as a service to web email or phone/IoT clients. One App covers this for you.
+- **Scalability:** Wrapped within an application that scales across 1,000's of engineers and multiple units of an enterprise website.
+- **Reusability:** Modules built as independent user experiences can be easily reused by multiple applications.
+
+
+# Architectural Overview
+
+The One App ecosystem draws a clear line between the App and the React components that make up the user interface. It allows for the React component tree to be split up into modular pieces so that they can be developed and deployed independently. We refer to these modular pieces as… **Modules**. It is an overloaded term, but we wanted to really drive home the point that when thinking about how to design your Modules, they should be as independent and self-contained as possible. The App is responsible for setting up the global redux store with middleware and reducers that all Modules will have access to, and kicking off the rendering of these Modules. [Holocron](https://one-amex-docs.americanexpress.com/en-us/holocron/api/) is the integral library that enables the App and Modules to be separate during development and come back together during production.
+
+## How does it work?
+
+Holocron maintains an in-memory registry of Modules that can be updated dynamically without requiring a server restart. The idea is that an application can update Holocron’s module registry whenever a new Holocron Module is to be added to the application’s runtime. The end result is an application that can have **React components updated/added to it at runtime**.
+
+One App is the Node.js server that acts as the orchestrator and stitches all your modules together as well as Server-Side Render your application.
+
+## In Production
+
+Holocron modules are bundled and their static assets deployed to a CDN. The One App Server instance polls the module map periodically for updates to the module versions as well as new modules recently deployed.
```

Comment:
Should we mention that we don't provide the CDN? I don't want people to get confused about that.

Reply:
Not sure what the confusion could be, we provide a React framework not a CDN... below we mention that modules can be deployed to any CDN